subprojects/core/src/integTest/groovy/org/gradle/initialization/CommandLineArgDeprecationIntegrationTest.groovy

```diff
@@ -32,8 +33,29 @@ class CommandLineArgDeprecationIntegrationTest extends AbstractIntegrationSpec {
 
         where:
         issue                                          | deprecatedArgs        | message
-        'https://github.com/gradle/gradle/issues/1425' | '--recompile-scripts' | StartParameterBuildOptions.RecompileScriptsOption.DEPRECATION_MESSAGE
-        'https://github.com/gradle/gradle/issues/3077' | '--no-rebuild'        | StartParameterBuildOptions.NoProjectDependenciesRebuildOption.DEPRECATION_MESSAGE
-        'https://github.com/gradle/gradle/issues/3077' | '-a'                  | StartParameterBuildOptions.NoProjectDependenciesRebuildOption.DEPRECATION_MESSAGE
+        'https://github.com/gradle/gradle/issues/1425' | '--recompile-scripts' | '--recompile-scripts has been deprecated and is scheduled to be removed in Gradle'
```

Comment:
:x: Instead of listing the issues as unused parameters, I'd just use the `Issue` annotation.

    @Issue(["https://github.com/gradle/gradle/issues/1425", "https://github.com/gradle/gradle/issues/3077"])

Reply:
Actually this can clearly show which issue refer to the deprecated options. So I prefer to keep them.